README.md

```diff
@@ -5,9 +5,9 @@ This example shows how to create a bootloader.
 To read more about the bootloader, please visit [bootloader tutorial](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/bootloader/).
 
 ## Required hardware
-* A supported board - [u-blox EVK-ODIN-W2](https://developer.mbed.org/platforms/ublox-EVK-ODIN-W2/), [Nucleo F429ZI](https://developer.mbed.org/platforms/ST-Nucleo-F429ZI/) or [K64F](https://developer.mbed.org/platforms/FRDM-K64F/).
-* CI test shield.
+* A supported target - [u-blox EVK-ODIN-W2](https://developer.mbed.org/platforms/ublox-EVK-ODIN-W2/), [Nucleo F429ZI](https://developer.mbed.org/platforms/ST-Nucleo-F429ZI/) or [K64F](https://developer.mbed.org/platforms/FRDM-K64F/).
 * SD card.
+* (If no onboard SD card port is present) Shield or breakout board connecting the SD card to the SPI pins.
```

Comment:
That's not the issue. I mean the second bullet says "you need a supported target (K64F, NUCLEO_F429ZI, UBLOX_ODIN)" and the third bullet would say "a target with an SD card". It sounds like you need two targets for this example.